lib/beamchmark/formatters/console.ex

```diff
@@ -85,11 +90,25 @@ defmodule Beamchmark.Formatters.Console do
     measurements_diff.context_switches)}
 
     #{entry_header("CPU Usage Average")}
-    #{format_percent_diff(measurements.cpu_info.average_all,
+    #{format_cpu_average(measurements.cpu_info.average_all,
     measurements_diff.cpu_info.average_all)}
+
+
+    #{entry_header("CPU Usage Per Core")}
+    #{format_cpu_by_core(measurements.cpu_info.average_by_core,
+    measurements_diff.cpu_info.average_by_core)}
     """
   end
 
+  defp format_cpu_average(cpu_average_new, cpu_average_diff) do
```

Comment:
In other cases we wrote something like 

```suggestion
  defp format_cpu_average(cpu_average, cpu_average_diff) do
```
so let's keep the convention :slightly_smiling_face:   

Please check other places too